compiler-rt/lib/radsan/radsan_user_interface.cpp

```diff
@@ -32,23 +33,20 @@ std::function<OnErrorAction()> createErrorActionGetter() {
       return OnErrorAction::Continue;
   };
 
-  auto user_mode = __sanitizer::GetEnv("RADSAN_ERROR_MODE");
-  if (user_mode == nullptr) {
-    return exit_getter;
-  }
+  const char* user_mode = radsan::flags()->error_mode;
```

Comment:
Another new use of this system with our custom flag

Reply:
Yeah this is nice!